lib/a.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+class A
+end
```

Comment:
Zeitweirk was [complaining](https://github.com/invalidusrname/movie-collector/actions/runs/7072514884/job/19251451151#step:7:12). It was expecting this file a constant A